composer.json

```diff
@@ -26,6 +30,7 @@
         "geoip2/geoip2": "If you are going to use the GeoIP2DatabaseProvider.",
         "symfony/stopwatch": "If you want to use the TimedGeocoder"
     },
+    "minimum-stability": "beta",
```

Comment:
Sadly no, because the discovery depends on puli which is in beta state currently :cry: